academic/pyCRAC/pyCRAC.info

```diff
@@ -1,12 +1,12 @@
 PRGNAM="pyCRAC"
-VERSION="1.5.1"
+VERSION="1.5.2"
 HOMEPAGE="http://sandergranneman.bio.ed.ac.uk/pycrac-software"
-DOWNLOAD="https://pypi.python.org/packages/source/p/pyCRAC/pyCRAC-1.5.1.tar.gz \
-          https://git.ecdf.ed.ac.uk/sgrannem/crac_pipelines/-/archive/b2647bae8c515fa3a790db3459342adb826a0a6a/crac_pipelines-b2647bae8c515fa3a790db3459342adb826a0a6a.tar.gz"
-MD5SUM="f634167e03ef26213efb4400671dbc9c \
-        dc396811a0df2c83cb52681c94e8cb69"
+DOWNLOAD="https://pypi.python.org/packages/source/p/pyCRAC/pyCRAC-1.5.2.tar.gz \
+          https://git.ecdf.ed.ac.uk/sgrannem/crac_pipelines/-/archive/master/crac_pipelines-master.tar.gz"
```

Comment:
This needs to point to a commit, otherwise your md5sum will not match as soon as another commit is made to master.